### PR TITLE
scaleway_organization_info: sanity checks

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Fix sanity checks.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
plugins/modules/scaleway_organization_info.py

### CODE REVIEW

File: plugins/modules/scaleway_organization_info.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 scaleway_organization_info:
   description: Response from Scaleway API
   returned: success
-  type: complex
+  type: list
```
Comment (felixfontein) [inline plugins/modules/scaleway_organization_info.py:47]:
```suggestion
  type: list
  elements: dict
```
Always add `elements` to `type: list` :)

Comment (russoz) [reply plugins/modules/scaleway_organization_info.py:47]: I wasn't sure about that, but yeah, I think it is likely so.

Comment (felixfontein) [reply plugins/modules/scaleway_organization_info.py:47]: If it's not, then the sample is wrong :)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**